style.css

```diff
@@ -306,5 +312,22 @@ input {
   border-radius: 8px;
   padding: 12px;
   align-self: flex-start;
-  margin-left: 30px;
+}
+
+.f-list {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  justify-content: center;
+  margin: 0 auto;
+}
+
+form {
```

Comment:
- [ ] [REQUIRED] Please, make sure that your design is similar to the Figma Design. The Border color for the `input` fields and the Footer section is not appropriate.

Your Design | Figma Design
------------|--------------
![image](https://user-images.githubusercontent.com/60979926/172596767-99dcaf57-d496-4f5f-8620-fb55654a4f50.png) | ![image](https://user-images.githubusercontent.com/60979926/172597076-7b83d2bc-4a39-4db7-a99e-2533c2f1c649.png)